src/pandamonium/pandamon.py

```diff
@@ -234,7 +234,6 @@ def getstatus(task, args):
                 s=status_color,
                 t=task['taskname'],
                 j=task['jeditaskid'],
-                i=task['reqid'],
```

Comment:
I'm removing this instance here even though `j=task['jeditaskid']` already exists as if `reqid` isn't useful then it should be ignored. Not sure if this was needed in output parsing though, @dguest?

Reply:
won't the dantrim option fail if you remove this? It requests `i` in the format string on line 220. I don't think we can remove an option without breaking some people's scripts, and since this is just the @dantrim option anyway we should leave it to him to add things as he sees fit.

Reply:
go ahead with what works best @dguest @matthewfeickert, i have not used panda in a long while so i am not hip to the latest needs

Reply:
But it's the @dantrim option. It's not the "show other stuff option", or the "we can change it whenever we want" option. It serves Danny Joseph Antrim and no one else, and he alone can change it.

Reply:
And he alone can change it. So say we all. :)